processes-endpoint.md

```diff
@@ -140,32 +140,144 @@ In a future version, a solution supporting a live-feed can also be included.
+        },
+        "format": {
+          "href": "/api/core/bitstreams/e2343390-0b1b-4397-be3d-66fed243a436/format"
+        },
+        "self": {
+          "href": "/api/core/bitstreams/e2343390-0b1b-4397-be3d-66fed243a436"
+        }
+      },
+      "_embedded": {
+        "format": {
+          "id": 1,
+          "shortDescription": "CSV",
+          "description": "Comma-separated values",
+          "mimetype": "text/csv",
+          "supportLevel": 0,
+          "internal": false,
+          "extensions": null,
+          "type": "bitstreamformat",
+          "_links" : {
+            "self" : {
+              "href": "/api/core/bitstreamformats/27"
+            }
+          }
+        }
+      }
     },
     "mapfile" : {
-      "href" : "/api/system/processes/4/files/mapfile"
+      "id": "e2343390-0b1b-4397-be3d-66fed243a436",
+      "uuid": "e2343390-0b1b-4397-be3d-66fed243a436",
+      "name": "map.txt",
+      "handle": null,
+      "metadata": {
+        "dc.title": [
+          {
+            "value": "map.txt",
+            "language": null,
+            "authority": null,
+            "confidence": -1,
+            "place": 0
+          }
+        ],
+        "dspace.process.filetype": [
+          {
+            "value": "mappingFile",
+            "language": null,
+            "authority": null,
+            "confidence": -1,
+            "place": 0
+          }
+        ]
+      },
+      "bundleName": null,
+      "sizeBytes": 479381,
+      "checkSum": {
+        "checkSumAlgorithm": "MD5",
+        "value": "64cdb4b4dd01b4a4cbb41e6d879fe996"
+      },
+      "sequenceId": null,
+      "type": "bitstream",
+      "_links": {
+        "content": {
+          "href": "/api/core/bitstreams/e2343390-0b1b-4397-be3d-66fed243a436/content"
+        },
+        "format": {
+          "href": "/api/core/bitstreams/e2343390-0b1b-4397-be3d-66fed243a436/format"
+        },
+        "self": {
+          "href": "/api/core/bitstreams/e2343390-0b1b-4397-be3d-66fed243a436"
+        }
     },
-    "zipfile" : {
-      "href" : "/api/system/processes/4/files/zipfile"
+      "_embedded": {
+        "format": {
+          "id": 1,
+          "shortDescription": "Text",
+          "description": "Plain Text",
+          "mimetype": "text/plain",
+          "supportLevel": 0,
+          "internal": false,
+          "extensions": null,
+          "type": "bitstreamformat",
+          "_links": {
+            "self": {
+              "href": "/api/core/bitstreamformats/6"
+            }
+          }
+        }
+      }
     }
   }
 }
 ```
 
-## Execution File Output
-**GET /api/system/processes/<:process-id>/files/<:file-name>**
-
-This endpoint will let an administrator download an output file created by a process. If the file is found, it will be presented as a download. This endpoint will support "Range" HTTP headers so that downloads can be paused and resumed.
+## Execution File Output List (type filter)
```

Comment:
We should not call this a "type filter" or even a "List", as both of these imply multiple files might be returned from this endpoint.  Let's rename this to be something like:
`Execution File Output (using type identifier)`

That helps to clarify that Type is an *identifier* here, and only *one* file will be returned based on the type.

Reply:
Made the change.